cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```diff
@@ -222,8 +222,35 @@ parameters:
 - name: CS_SERVICE_PRINCIPAL_CREDS_BASE64
   description: Base64 encoded service principal credentials for CS. This is temporary for P1
   required: true
+- name: SERVICE_KEYVAULT_NAME
+  description: The name of the Key Vault containing keys, secrets and certificates used to support services running on the cluster service.
+  value: "service-kv-aro-hcp-dev"
+- name: TENANT_ID
+  description: The Tenant ID of Key Vault.
+  value: "64dc69e4-d083-49fc-9569-ebece1dd1408"
```

Comment:
i would prefer to not provide a default here targeting a specific tenant

Reply:
I would prefer that too, @JameelB wdyt?